Dockerfile

```diff
@@ -1,23 +1,18 @@
-FROM maven:3.6-jdk-11
-ENV MAXWELL_VERSION=1.33.0 KAFKA_VERSION=1.0.0
+#Docker image of MaxWell Can Support Amd64 and Arm64/v8
+FROM openjdk:11-slim
+#FROM openjdk:11
+ENV MAXWELL_VERSION=1.33.0
+ENV MAXWELL_FILE maxwell-1.33.0.tar.gz
+ENV MAXWELL_DOWNLOAD_PATH "https://github.com/zendesk/maxwell/releases/download/v1.33.0/maxwell-1.33.0.tar.gz"
```

Comment:
This creates a circular dependency. This would mean we need to make a public release in order to deploy docker. This doesn't allow for iteration / changes in a local environment like the current one does.

Reply:
Yes. But I think there is a another idea to resolve the docker support Arm64.  That is build a maven image to support Arm64 and then use the old dockefile. But I think it will be little complex.

Reply:
It looks like maven has an openjdk-11-slim version: https://github.com/carlossg/docker-maven/blob/bdffb5117c33476d554325d8efe5866306004b99/openjdk-11-slim/Dockerfile.

https://hub.docker.com/_/maven?tab=description:
```
3.8.1-jdk-11-slim, 3.8-jdk-11-slim, 3-jdk-11-slim, 3.8.1-openjdk-11-slim, 3.8-openjdk-11-slim, 3-openjdk-11-slim
```


Does that work?

Reply:
It looks like arm64 with the regular non-slim version works for arm64v8 too.

https://hub.docker.com/r/arm64v8/maven/

Looking at the git history, arm84v8 has been supported since 3.6.1.

https://github.com/docker-library/official-images/commit/ed6120b4d885ace6fb7d400021371bfcbcbe40c0#diff-79f56c4595d485f2c306310171f04dcd7d027f418e1a6bee7186dedd2fbedb7c